apps/vr-tests-react-components/src/stories/Positioning.stories.tsx

```diff
@@ -273,28 +273,116 @@ const VerticalOverflow = () => {
 const HorizontalOverflow = () => {
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26730&commitId=3b4871862b047cdee6ff768dbca42da18d6a158e&env=prod)

✅ There was 0 screenshots added, 1 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pill.Pill Selectable Example.chromium.png|0| Removed|

</details>


<div id="vrtCommentFluentUI-v0"/>